projects/dsp-ui/src/lib/action/pipes/array-transformation/sort-by.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'dspSortBy'
+})
+export class SortByPipe implements PipeTransform {
+
+    /**
+     * compares value by value and sorts by alphabetical order
+     */
+    transform(array: Array<any>, args: string): Array<any> {
```

Comment:
I think an parameter should not be called  `array`.

Reply:
Is an array of objects expected and `args` is the key in an object?
If so, in case the key does not exist, it would be `undefined`, not `null`.

Reply:
better check that the param is an array, not just that it is not `undefined`

Reply:
parameter has been renamed to `value` and the check now checks for an instance of an array in 84557d0